src/main/scala/strawman/collection/Iterable.scala

```diff
@@ -145,6 +156,14 @@ trait IterableOps[+A] extends Any {
   *  as the one they are invoked on.
   */
 trait IterableMonoTransforms[+A, +Repr] extends Any {
+  def filter(p: A => Boolean): Repr
```

Comment:
This operation shouldn't be here. If you filter a range, it may stop being a range.